indigo/analysis/refs/base.py

```diff
@@ -246,25 +240,50 @@ class SectionRefsFinderENG(BaseSectionRefsFinder):
         TODO: match paragraphs
         TODO: match multiple sections
         TODO: match ranges of sections
-
     """
 
     # country, language, locality
     locale = (None, 'eng', None)
 
-    section_re = re.compile(
-        r'''(?P<ref>
-        (?P<section_ref>
-        \b[sS]ections?\s+
-        (?P<num>\d+[A-Z]*)
+    ref_re = re.compile(
+        r'''
+        (?P<ref>
+          \b[sS]ections?\s+
+          (?P<num>\d+[A-Z]*)
         )
         (?P<subsection_ref>\s*\(\d+[A-Z]*\))?
```

Comment:
I think these named groups add complexity unnecessarily, because they aren't used at all.

Reply:
I guess it's a bit early as we're not using them yet but I thought it might make it more readable, and they should come in handy once we start matching below section level?

Reply:
This is a bit like writing a method or function that you think you may use, but don't know quite yet. It's better to write it later, when you actually need it, than to guess what you'll need in future. In most cases, that guess will be wrong.